app/models/ability.rb

```diff
@@ -32,6 +32,7 @@ def initialize(user)
     user ||= User.new # guest user (not logged in)
 
     basic(user)
+    canvas_oauth_user(user) if user.role?("canvas_oauth_user")
```

Comment:
Want to add this method so this doesn't break?

Reply:
done